src/sentry/static/sentry/app/utils/projects.tsx

```diff
@@ -394,6 +394,10 @@ class Projects extends React.Component<Props, State> {
 
 export default withProjects(withApi(Projects));
 
+export async function fetchProjectsCount(api: Client, orgId: string) {
+  return await api.requestPromise(`/organizations/${orgId}/projects-count/`);
+}
```

Comment:
Can we move this to `app/actionCreators/projects`? Would like to keep this file only related to `<Projects>`